sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -309,5 +447,15 @@ public async Task GetTokenForTeamsUserWithWrongUserObjectIdShouldThrow()
             }
             Assert.Fail("An exception should have been thrown.");
         }
+
+        private bool IsTokenExpirationValid(TimeSpan expectedTokenExpiration, DateTimeOffset tokenExpiresAfter)
```

Comment:
I would add a third parameter `int allowedDeviation`, otherwise the method name is a bit misleading.

Reply:
Done.